src/exercises/adapt-c/Makefile

```diff
@@ -0,0 +1,14 @@
+CC=clang
+
+ARCH!=uname -n
+.if ${ARCH} == "cheribsd-morello-purecap"
+CFLAGS=-g -O2 -target aarch64-unknown-freebsd -mno-relax -Wall -Wcheri
+CFLAGS_CHERIABI=-march=morello+noa64c
+CFLAGS_CAPABI=-march=morello+c64 -mabi=purecap -Xclang -morello-vararg=new
```

Comment:
??????

CheriABI *is* "Cap ABI", whatever that is. You're doing non-CHERI vs CHERI.